### PR TITLE
Title: Feat/customuser

### PR DESCRIPTION
**Pull request by SiddonsD**
Body:
- created custom user model, serializers and views
- add token auth
- add skill and interests to custom user model

Tested the following endpoints:
POST /api-token-auth/
image

GET/users/
image

POST /user/register/
image

GET /users/<user_id>/
image

### CODE REVIEW

File: exception_al/user/urls.py [hunk 1]
```diff
@@ -3,5 +3,7 @@
 
 urlpatterns = [
     path('users/', views.CustomUserList.as_view()),
-    path('users/<int:pk>/', views.CustomUserDetail.as_view()),
+    path('user/register', views.CustomUserRegister.as_view()),
+    path('user/<int:pk>/', views.CustomUserDetail.as_view()),
+    path('api-token-auth/', views.CustomAuthToken.as_view(), name='api_token_auth')
```
Comment (suanTech) [inline exception_al/user/urls.py]: This line of code can be moved to the project's url file!

**Review by suanTech — COMMENTED** [review]
Looks great!! Just need to fix duplicate api_token_auth path in project's urls.py and User's urls.py. I think the one in User's urls.py can be moved into the project's urls.py.

File: exception_al/user/urls.py (hunk 1 above)
Comment (SiddonsD) [reply exception_al/user/urls.py]: Thank you for checking  @suanTech. I've removed that duplicate url from the users/urls.py file.

Comment (SiddonsD) [reply exception_al/user/urls.py]: image

**Review by suanTech — APPROVED** [review]
Looks great 🤩

**Pull request merged**